app/models/custom-item-metadata-record.ts

```diff
@@ -0,0 +1,28 @@
+import { AsyncBelongsTo, attr, belongsTo } from '@ember-data/model';
+import GuidModel from 'ember-osf-web/models/user';
+
+import OsfModel from './osf-model';
+
+export interface Funder {
+    funderName: string;
+    funderIdentifier: string;
+    funderIdentifierType: string;
+    awardNumber: string;
+    awardUri: string;
+    awardTitle: string;
```

Comment:
thinking these should be `?`-optional, too -- will be null or undefined if they're not already set to some value